package.json

```diff
@@ -19,10 +19,14 @@
     "@polymer/iron-flex-layout": "^3.0.0-pre.20",
     "@polymer/iron-test-helpers": "^3.0.0-pre.20",
     "wct-browser-legacy": "^1.0.1",
-    "@webcomponents/webcomponentsjs": "^2.0.0"
+    "@webcomponents/webcomponentsjs": "^2.0.0",
+    "@polymer/gen-typescript-declarations": "^1.5.0",
+    "lodash": "^3.0.0"
```

Comment:
Why does this need lodash?

Reply:
So @e111077, @azakus, and I were just discussing this and thought that it might make sense for wct-browser-legacy to include [the files it needs to synchronously load](https://github.com/Polymer/tools/blob/0706809c7fba09d3923ff464e3038b93f4307e2c/packages/wct-browser-legacy/browser.js#L212-L234) in the `bundledDependencies` field of its `package.json`. That way, their location would be guaranteed to be in wct-browser-legacy's node_modules folder. Then, wct-browser-legacy would just need to figure out where it was to be able to know where all of those files are (maybe using `document.currentScript`'s `src` + `baseURI`).